frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -843,6 +844,29 @@ function ReaderHighlight:onHoldRelease()
     return true
 end
 
+function ReaderHighlight:onCycleHighlightAction()
+    local next_actions = {
+        highlight = "translate",
+        translate = "wikipedia",
+        wikipedia = nil
+    }
+    local current_action = G_reader_settings:readSetting("default_highlight_action")
+    if not current_action then
+        G_reader_settings:saveSetting("default_highlight_action", "highlight")
+        UIManager:show(Notification:new{
+            text = _("Default highlight action changed to \"highlight\""),
+            timeout = 1,
+        })
+    else
+        local next_action = next_actions[current_action]
+        G_reader_settings:saveSetting("default_highlight_action", next_action)
+        UIManager:show(Notification:new{
+            text = T(_("Default highlight action changed to \"%1\""), (next_action or "default")),
```

Comment:
The house style is single quotes. I should write that down somewhere more formally.

```suggestion
            text = T(_("Default highlight action changed to '%1'."), (next_action or "default")),
```